Tests/interop/net/field/test_instance_fields.py

```diff
@@ -298,74 +300,72 @@ def f24(): t.__dict__['InstanceSimpleInterfaceField'].__set__(v, ClassImplementS
                 with warning_trapper() as wt:
                     f()
                 AssertFieldWarnings(wt)
-        else: 
+        else:
             for f in funcs: f()
             self._test_verify(o)
```

Comment:
I was fixing all the test errors.

The list comprehensive has its own scope, so that `f*` functions are not in the locals for it. As a result, it will fail to find those functions. Using a for-loop could avoid it.

I omit this one because it is never called and doesn't fail the build. This “test” method was used to define some functions, but not executed. I changed that to call the “test” method directly.

Reply:
Good catch! Looks like these tests stopped running when it got converted to a unittest.